it/thrift0.9.1/src/test/java/java/com/linecorp/armeria/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * All sub-packages of this package contain generated files by Thrift Compiler 0.9.1.
+ * The generated code is intentionally added because Thrift compiler 0.9.3 is not compatible with
+ * libthrift 0.9.1.
```

Comment:
Note for reviewers.
Only this file and `it/thrift0.9.1/build.gradle` are created by hand. 😆